<!DOCTYPE html>
<html>
	<head>
		    <meta charset="utf-8">
		<title>CVE-2020-12850 - Vulnerability Data</title>
		    <link rel="shortcut icon" type="image/png" href="/favicon.ico" />
	</head>
<body>
<h2>Vulnerability Data</h2>
<h1>CVE-2020-12850</h1>
<p>The following vulnerability applies only to the Pydio Cells Enterprise OVF version 2.0.4. Prior versions of the Pydio Cells Enterprise OVF (such as version 2.0.3) have a looser policy restriction allowing the “pydio” user to execute any privileged command using sudo. In version 2.0.4 of the appliance, the user pydio is responsible for running all the services and binaries that are contained in the Pydio Cells web application package, such as mysqld, cells, among others. This user has privileges restricted to run those services and nothing more.</p>

<dl>
	<dt>Published</dt>
	<dd>2020-06-10</dd>

	<dt>Modified</dt>
	<dd>2020-06-11</dd>
</dl>

<h3>References</h3>
<ul>
	<li><a href="http://packetstormsecurity.com/files/158002/Pydio-Cells-2.0.4-XSS-File-Write-Code-Execution.html">MISC:http://packetstormsecurity.com/files/158002/Pydio-Cells-2.0.4-XSS-File-Write-Code-Execution.html</a></li>
<li><a href="https://www.coresecurity.com/advisories">MISC:https://www.coresecurity.com/advisories</a></li>
<li><a href="https://www.coresecurity.com/core-labs/advisories/pydio-cells-204-multiple-vulnerabilities">MISC:https://www.coresecurity.com/core-labs/advisories/pydio-cells-204-multiple-vulnerabilities</a></li>
</ul>

<footer>
	<p><a href="https://reconmap.org">Track security vulnerabilities</a> with <strong>Reconmap</strong>.</p>
</footer>
